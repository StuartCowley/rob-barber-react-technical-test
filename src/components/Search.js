import React from "react";
import { useState } from "react";
import "../styles/search.css";

function Search() {
  return (
    <>
    <input className="search-field" type="text"></input>
    </>
  )
}

export default Search;