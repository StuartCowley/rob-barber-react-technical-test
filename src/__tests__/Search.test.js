import { render, screen } from "@testing-library/react";
import Search from "../components/Search";

describe("search", () => {
   const { asFragment } = render(<Search />)
   
   it("renders correctly", () => {
      expect(asFragment()).toMatchSnapshot();
   });
   
   });