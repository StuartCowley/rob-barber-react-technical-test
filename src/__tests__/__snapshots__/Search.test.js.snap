// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search renders correctly 1`] = `
<DocumentFragment>
  <input
    class="search-field"
    type="text"
  />
</DocumentFragment>
`;
